addons/im_livechat/models/discuss_channel.py

```diff
@@ -315,3 +315,10 @@ def _action_unfollow(self, partner=None, guest=None, post_leave_message=True):
             # sudo: discuss.channel - last operator left the conversation, state must be updated.
             channel_sudo.livechat_active = False
             self._bus_send_store(self, "livechat_active")
+
+    def add_members(
+            self, partner_ids=None, guest_ids=None, invite_to_rtc_call=False, post_joined_message=True
```

Comment:
We might as well just take `**kwargs` as we don't use any of those arguments and it would make the override more robust against changes.

Reply:
We can't change method signature in overrides. There is a new test lint for that too.

Reply:
Still, less maintenance when updating original method and why bother listing them twice if we don't care about them?

Reply:
Literally we can't do it anymore as a test will fail.

And it's a bad practice because it means the method can't be called with positional parameters once livechat is installed, even though the base method can.

Reply:
https://github.com/odoo/odoo/pull/203296

Reply:
There is no longer any question about maintenance or robustness as it is automatically checked, we can't do the mistake of forgetting anymore.
Just copy paste all the params from the original and be done with it.